### PR TITLE
fix: broken list format and typo fixes on workshop

### PR DESCRIPTION
**Pull request by tancnle**
# Description

- Fix broken list format on D/HD Workshop doc
- Fix some small typos
- Re-order sections according to the original [doc](https://deakin365.sharepoint.com/:w:/r/sites/ThothTech2-Leadership/_layouts/15/Doc.aspx?sourcedoc=%7B8E4ED901-DC53-4771-A4C7-8780CEFA89FA%7D&file=D%20HD%20Workshop%20Notes.docx&action=default&mobileredirect=true)

## Type of change

- [x] Documentation (update or new)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: docs/learning/training/distinction-high-distinction-workshop.md [hunk 1]
```diff
@@ -9,27 +45,28 @@ team's work.
 
 At this level, you will assist in building the pieces needed by your team to complete the puzzle. It
 would help if you had support to complete your work to an acceptable standard or get the tasks done
-on time.
+on time. Luckily, you have a team that will provide you with that support; make sure you don't let
+them down.
```
Comment (ghost) [inline docs/learning/training/distinction-high-distinction-workshop.md]: **Suggestion:** removes contraction. Sounds more positive and encouraging

```suggestion
on time. Luckily, you have a team that will provide you with that support; make sure you are doing
them proud.
```

File: docs/learning/training/distinction-high-distinction-workshop.md [hunk 2]
```diff
@@ -9,27 +45,28 @@ team's work.
...
+At this level you will be building the pieces needed by your team to complete the puzzle. You work
+at this level if you are a responsible team member. You can follow instructions from your lead and
 deliver the designated work at an acceptable standard in the allocated timeframe. You must show:
 
-- Clear contributions to your project
-- Be engaged and Present
-- Responsibility
+- Clear contributions
+- Be engaged and present
+- Responsible
```
Comment (ghost) [inline docs/learning/training/distinction-high-distinction-workshop.md]: Reword: related to above
```suggestion
- Show clear contributions
- Be engaged and present
- Be responsible
```

File: docs/learning/training/distinction-high-distinction-workshop.md [hunk 3]
```diff
@@ -9,27 +45,28 @@ team's work.
...
 
 ### Credit
 
 Show you are a responsible team member to receive a credit.
 
-At this leve you will be buliding the pieces needed by your team to complete the puzzle. You work at
-this level if you are a responsible team member. You can follow instructions from your lead and
+At this level you will be building the pieces needed by your team to complete the puzzle. You work
+at this level if you are a responsible team member. You can follow instructions from your lead and
 deliver the designated work at an acceptable standard in the allocated timeframe. You must show:
```
Comment (ghost) [inline docs/learning/training/distinction-high-distinction-workshop.md]: reword:
```suggestion
deliver the designated work at an acceptable standard in the allocated timeframe. You must:
```

File: docs/learning/training/index.md [hunk 4]
```diff
@@ -5,8 +5,6 @@ technical development.
 
 A list of training can be seen below:
 
-## Training
-
 - Installing npm
 - Product Epics
 - User Stories
```
Comment (ghost) [inline docs/learning/training/index.md]: Should these be linked? and would it be better to just link to the training folder?

**Review by ghost — APPROVED** [review]
Nice catch Tan!.
Have left some comments for your review., happy to merge at your discretion.

File: docs/learning/training/index.md (hunk 4 above)
Comment (tancnle) [reply docs/learning/training/index.md]: Thanks, @talzid. From what I understand, we do not have these resources built yet. I could not find them anywhere in the handbook repo 🤔

Comment (ghost) [reply docs/learning/training/index.md]: Good point. Might be worth linking npm to the contribution file, user stories to the QA section, and removing product epics?

> would it be better to just link to the training folder? 

My mistake didn't realise that's where this was

Comment (tancnle) [reply docs/learning/training/index.md]: > user stories to the QA section, and removing product epics?

Good idea. I will link epics and user stories to the QA section

> npm to the contribution file

Will drop the NPM installation for now. I will bring the one Morgaine created in the documentation repo over here.

**Pull request merged**